packages/spruce-skill/.storybook/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
In ai-web I'm updating this to:

```
{
	"extends": "../tsconfig.json",
	"compilerOptions": {
		"esModuleInterop": true
	}
}
```

Use the root config, plus add the interop option storybook needs

Reply:
@gcpantazis not sure why, but I have to also include `"jsx": "react"` here otherwise storybook gets angry.